manifests/filebeat.pp

```diff
@@ -1,7 +1,7 @@
 # Wazuh App Copyright (C) 2019 Wazuh Inc. (License GPLv2)
 # Setup for Filebeat
 class wazuh::filebeat (
-  $filebeat_elasticsearch_ip = 'localhost',
+  $filebeat_elasticsearch_ip = '<Put Elasticsearch IP>',
```

Comment:
Rename to <<YOUR_ELASTICSEARCH_IP> please

Reply:
Hi Jose!

Thanks for your attention. Check the changes [here](https://github.com/wazuh/wazuh-puppet/commit/1301d408b350d150f40ea1e58dbace59119cc99a).

Kind regards,

Rshad